packages/terra-functional-testing/lib/terra-dev-site/doc/asset-server-service/AssetServerService.doc.mdx

```diff
@@ -0,0 +1,199 @@
+# Asset Server Service
+
+The asset server service initializes a server to serve compiled assets to the testing environment.
+
+## Installation
+
+The asset server service should be installed as a development dependency.
+
+```bash
+npm install --save-dev @cerner/terra-functional-testing
+```
+
+## Configuration
+
+To use the service it must be added to the services array in the `wdio.conf.js` file.
+
+```js
+// wdio.conf.js
+const AssetServerService = require('@cerner/terra-functional-testing/lib/services/wdio-asset-server-service');
+
+export.config = {
+    // ...
+    services: [
+        [AssetServerService, {
+            // Asset server service options here.
+            // ...
+        }]
+    ],
+    // ...
+};
+```
+
+## Options
+
+### config
```

Comment:
I think we should call this out as `webpackConfig`.

Reply:
Renamed to `webpackConfig`:

https://github.com/cerner/terra-toolkit/pull/417/commits/c6011cf084d1e24c1a1df7097dd293f21552a0ae